### ISSUE TITLE
Width is smaller when text has <b> or <strong> tags

### ISSUE THREAD

**Issue by abdatta** [DesertFoxNV/ngx-autosize-input#6]
The width is smaller and text gets hidden when the input has `<b>` or `<strong>` tags. Please fix this issue.

**Comment by joshuawwright** [comment]
Can you show me an example of the html to generate the problem.  Is it like `<b> <input></input> </b>`? Ore are you just setting the font weight?

**Comment by abdatta** [comment]
```
       <strong>
          <input  AutoSizeInput
                  type="text"
                  [disabled]="saving"
                  [ngModel]="tile['name']"
                  (ngModelChange)="change($event,i,'name')">
       </strong>
```
This is an extract directly from my code. This gives slightly smaller width that partially covers the text. The `<strong>` tags are the problem. Without them it works fine. `<b>` tags produce same problem. I don't exactly remember now, but I think I've tried setting the `font-weight` also. You may check this.

**Comment by joshuawwright** [comment]
@abdatta This should now be fixed in version 2.1.1. Strong and bold tags will not affect the input. Instead you will need to use font-weight inline or in a stylesheet.

**Comment by abdatta** [comment]
The input width the proper now for `<strong>` and `<b>` tags. But it's larger than required for normal text.
Here's a screenshot. Note there are no trailing spaces.
![image](https://user-images.githubusercontent.com/20218826/44196113-af987b80-a158-11e8-9b3d-d9cb7b6f8524.png)

Attached code: 
Works Fine!  :
```
<strong>
          <input  AutoSizeInput
                  type="text"
                  [disabled]="saving"
                  [ngModel]="tile['name']"
                  (ngModelChange)="change($event,i,'name')">
</strong>
```
Extra Width !! : 
```
<input  AutoSizeInput
                  type="text"
                  [disabled]="saving"
                  [minWidth]="6"
                  [ngModel]="tile['more'][j]"
                  (ngModelChange)="change($event,i,'more',j)">
```

**Comment by joshuawwright** [comment]
@abdatta can you try setting `[includeBorders]=false`? Did that fix the problem? In the next version 2.2.0 I set the includeBorders default value to false.

**Comment by abdatta** [comment]
Works perfectly with version 2.2.0. Thanks! 👍
